server/controllers/gh.js

```diff
@@ -62,6 +62,23 @@ function getComments(req, res) {
     });
 }
 
+function notFoundPage(req, res){
+    makeIssueRequest(issuesRequestUrl).then(function(issues) {
+        var latestIssues =  issues.filter(function(issue) {
+                                var tr = Math.abs(moment(issue.created_at).diff(moment(), 'days')) <= 30;
```

Comment:
BTW, don't you think it's better to render just last 10 issues?
there might be 1000 or 0 issue in last 30 days and 10 is always 10 ;)